cloud/linode/loadbalancers.go

```diff
@@ -8,6 +8,7 @@ import (
 	"net/http"
 	"os"
 	"reflect"
+	"slices"
```

Comment:
@okokes-akamai i couldn't find a way to make this work on go 1.20 && 1.21 - should I default to using x/exp/slices until we get rid of 1.20 ?

Reply:
Well, we already build CCM with 1.21, so I'm not sure why we're testing this against 1.20. I wasn't happy about this back when I reviewed this PR: https://github.com/linode/linode-cloud-controller-manager/pull/147#discussion_r1413611112

Also, in about three weeks, `oldstable` will start pointing to 1.21, so we'll get rid of this automatically, but I'd rather drop the stable/oldstable altogether.

Reply:
will create a follow up oldstable stuff - keeping `x/exp/slices` for now